ckanext/dcat/converters.py

```diff
@@ -125,3 +139,36 @@ def ckan_to_dcat(package_dict):
         dcat_dict['distribution'].append(distribution)
 
     return dcat_dict
+
+
+def disallow_file_format(file_format):
+    if config.get('ckanext.format_filter.filter_type') == 'whitelist':
+        if file_format in get_whitelist():
+            return False
+        return True
+    elif config.get('ckanext.format_filter.filter_type') == 'blacklist':
+        if file_format in get_blacklist():
+            return True
+    return False
+
+
+def get_whitelist():
+    whitelist_string = config.get('ckanext.format_filter.whitelist', '')
+    return convert_to_filter_list(whitelist_string)
+
+
+def get_blacklist():
+    blacklist_string = config.get('ckanext.format_filter.blacklist', '')
+    return convert_to_filter_list(blacklist_string)
+
```

Comment:
The reason for having both blacklist and whitelist, is it for general purpose of some clients would want to use one or another?

Reply:
Users can define both a whitelist and blacklist, but the config `ckanext.format_filter.filter_type` will determine which is used. This'll let users more easily switch between a whitelist and blacklist id needed.